ci/ray_ci/docker_container.py

```diff
@@ -14,7 +14,7 @@
     "cu12.1.1",
 ]
 GPU_PLATFORM = "cu11.8.0"
-DEFAULT_PYTHON_VERSION = "3.8"
+DEFAULT_PYTHON_VERSION = "3.9"
```

Comment:
why is this declared twice here and in `builder_container.py`, and also `builder.py`? can we unify?